.github/workflows/staging.yml

```diff
@@ -0,0 +1,79 @@
+name: Deploy to staging when pushing on staging
+
+on:
+  push:
+    branches:
+      - staging
+      - feat/github-ci
```

Comment:
Why feat/github-ci ?

Reply:
It was in order to test the branch. We can remove it later